internal/integration/assert.go

```diff
@@ -63,9 +63,13 @@ func AssertListDetails[D ListDetailsMsg](t testing.TB, expected, actual D) {
 		return
 	}
 
-	gotCD := gotDetails.GetTimestamp().AsTime()
-	now := time.Now()
-	assert.WithinRange(t, gotCD, now.Add(-time.Minute), now.Add(time.Minute))
-
 	assert.Equal(t, wantDetails.GetTotalResult(), gotDetails.GetTotalResult())
+	// only check timestamp and sequence if result is greater than 0, as it could also be empty
```

Comment:
do we really return no date when there's no result?

Reply:
No, that's why "could be empty", but if the result is 0 than you have no knowledge on what contents should be in the ListDetails.